wgengine/magicsock/peermtu.go

```diff
@@ -110,3 +115,7 @@ func (c *Conn) UpdatePMTUD() {
 	c.peerMTUEnabled.Store(newStatus)
 	c.resetEndpointStates()
 }
+
+func isPMTUError(err error) bool {
+	return errors.Is(err, unix.EMSGSIZE)
```

Comment:
using `unix.EMSGSIZE` allocates, the pattern in `endpoint_default.go` shows using a global var and a more generic syscall constant.

https://github.com/tailscale/tailscale/pull/9004#discussion_r1300816537